building-web-forms/form-and-form-elements/README.md

```diff
@@ -0,0 +1,67 @@
+# Script
+
+In this video, we will learn about HTML Forms and HTML elements used within it to collect user input in web pages.
+
+The HTML `form` element is used to create an HTML form for user input. It is used as a container for elements such as text fields, checkboxes, radio buttons and submit buttons.
```

Comment:
The HTML `form` element is used to create an HTML form for user input. 

- This sentence is almost identical to previous one.

Reply:
This is just an explanation of the `form` tag itself. Keeping it as this is a reference to the element itself, unlike the previous text.